lib/daylight.cpp

```diff
@@ -2335,10 +2480,23 @@ bool Daylight::createBaseRadFiles(Control *model){
                 if(!wgRadModel->addRad(model->spaceDirectory()+model->geoDirectory()+model->windowGroups()[j].baseGeometry())){
                     return false;
                 }
-                for (int k=0;k<model->windowGroups()[j].glazingLayers().size();k++){
-//                    if(!wgRadModel->blackOutLayer(model->windowGroups()[j].glazingLayers()[k])){
-//                        return false;
-//                    }
+				std::vector<std::string> layers1 = model->windowGroups()[j].glazingLayers();
+				//std::string xs = toString(layers1[0]);
+
+
+                for (int k=0;k<model->windowGroups()[j].glazingLayers().size();k++){ 
+					//std::shared_ptr<RadPrimitive> RadFileData::glazingsearch( std::string name);
+					auto materialaddress = radModel.glazingsearch(model->windowGroups()[j].glazingLayers()[k]);
+
+					STADIC_LOG(Severity::Info, "SET ALIAS IS NEXT" + layers1[0]);
+
+					wgRadModel->setAlias(materialaddress, found);
+						//wgRadModel->setAlias(found, found);
+	std::string xsx = toString(wgRadModel->m_aliases.size()); 
```

Comment:
This is also fixed by adding an getter for the aliases vector. I've done that, so this can be changed to

    std::string xsx = toString(wgRadModel->aliases.size());